core/src/main/scala/cats/syntax/apply.scala

```diff
@@ -23,13 +23,8 @@ package cats
 package syntax
 
 trait ApplySyntax extends TupleSemigroupalSyntax {
-  implicit final def catsSyntaxApply[F[_], A](fa: F[A])(implicit F: Apply[F]): Apply.Ops[F, A] =
```

Comment:
Probably we need to keep this for bincompat, but we can remove the `implicit`

Reply:
If we remove `implicit` for the method itself, then usually it'd be better to remove `implicit` for the `F` parameter as well, i.e.:
```
final def catsSyntaxApply[F[_], A](fa: F[A])(F: Apply[F]): Apply.Ops[F, A]
```